README.md

```diff
@@ -31,35 +31,26 @@ The ```main``` branch of this repository contains samples configured for the lat
 
 The *ArcGIS Runtime SDK Samples app* has a *Target SDK* version of *13.0*, meaning that it can run on devices with *iOS 13.0* or newer.
 
-## Building samples using installed SDK
+## Building Samples Using Swift Package Manager
 
 1. **Fork** and then **clone** the repository
-    > Make sure to use the "recursive" option to ensure you get the **ArcGIS Runtime Toolkit** submodule
-    >
-    >`git clone --recursive [URL to Git repo]`
-    >
-    > If you've already cloned the repo without the submodule, you can load the submodule using 
-    >
-    >`git submodule update --init`
-1. **Install** the ArcGIS Runtime SDK for iOS to a central location on your Mac as described [here](https://developers.arcgis.com/ios/get-started)
-1. **Open** the `arcgis-ios-sdk-samples.xcodeproj` **project** file
 1. **Run** the `arcgis-ios-sdk-samples` app target
```

Comment:
Before this step, we need one that says open the project file. Something like - 

1. **Open** the `arcgis-ios-sdk-samples.xcodeproj` **project** file
> The project has been configured to use the swift package for [ArcGIS Runtime iOS Toolkit](https://github.com/Esri/arcgis-runtime-toolkit-ios). Xcode will automatically start downloading the latest toolkit, and in turn, the swift package for [ArcGIS Runtime iOS SDK](https://github.com/Esri/arcgis-runtime-ios) which the toolkit depends upon.